### DISCUSSION TITLE
Polish language version

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/commerce#2421] [Ideas]
Are there any plans for polish language version of commerce?

**Comment by angrybrad** [comment]
@piotrpog we manage our translations for Craft and Commerce on Crowdin: https://crowdin.com/project/craft-commerce

Our policy is if the community wants a new language and gets it to 100% translated, they'll we'll maintain that language going forward.

If that's something you want to contribute to, let me know and I'll make Polish a language on Commerce so you can contribute.
